localstack/services/route53resolver/models.py

```diff
@@ -106,3 +114,72 @@ def delete_firewall_rule(firewall_rule_group_id, firewall_domain_list_id):
     firewall_rule = get_firewall_rule(firewall_rule_group_id, firewall_domain_list_id)
     region_details.firewall_rules.get(firewall_rule_group_id, {}).pop(firewall_domain_list_id)
     return firewall_rule
+
+
+def get_resolver_query_log_config(id):
```

Comment:
It would be preferable to attach these util functions to the backend directly, i.e.:
```
class Route53ResolverBackend(RegionBackend):
    ...
    def get_resolver_query_log_config(self, id):
        ...
```

This will help us to unify the access to the backends, and avoid having too many calls to `Route53ResolverBackend.get()` spread over the codebase..

Can we make this change for the new util functions below, as well as the existing ones above?

Reply:
Chiming in here. I think this was due to a previous suggestion of mine since I wanted to keep the backends as lean as possible to avoid picking issues. But as discussed with @whummer, this should not be a big issue.

Reply:
Ah, ok good point @giograno . Either way is fine - we can also fix that up in a follow-up iteration, if and when we make the switch to unify the backend access protocol.. 👍

Reply:
Hey, It would be a clean approach to keep the backend related operation in the same class. I've updated it.